build.gradle.kts

```diff
@@ -42,3 +46,23 @@ spotless {
         ktlint("0.50.0")
     }
 }
+
+fun Project.configureSentry() {
+    extensions.getByType(SentryPluginExtension::class.java).apply {
+        includeProguardMapping = System.getenv()["CI"].toBoolean()
+                && !project.properties["skipSentryProguardMappingUpload"]?.toString().toBoolean()
+
+        tracingInstrumentation {
+            features.set(EnumSet.allOf(InstrumentationFeature::class.java) - InstrumentationFeature.FILE_IO)
+        }
+        autoInstallation.enabled = false
+        includeDependenciesReport = false
+        ignoredBuildTypes = setOf("debug")
+    }
+}
+
+rootProject.subprojects {
+    plugins.withId(rootProject.libs.plugins.sentry.get().pluginId) {
+        configureSentry()
+    }
+}
```

Comment:
I tried using idiomatic Gradle way, by convention plugins in `buildSrc`. The problem with this approach is that Sentry Gradle Plugin needs to have provided Android Gradle Plugin as dependency (they use `compileOnly`). Adding AGP to `buildSrc` creates conflicts as `buildSrc` dependencies are added to classpath - then Gradle cannot determine the desired version of AGP.

It's quickly becoming a mess for such a simple configuration, hence this small extension function and `subprojects` closure.